src/js/components/form-elements/v2/DateField.jsx

```diff
@@ -74,6 +75,7 @@ const DateField = ({
       tooltip={tooltip}
       errorMessage={errorMessage}
       button={button}
+      hideErrorMessage={hideErrorMessage}
```

Comment:
I think the error is hidden when the `errorMessage` is empty. Is this prop required?